aksel.nav.no/website/pages/eksempler/radio/error.tsx

```diff
@@ -1,14 +1,21 @@
+import { useState } from "react";
 import { Radio, RadioGroup } from "@navikt/ds-react";
 import { withDsExample } from "@/web/examples/withDsExample";
 
 const Example = () => {
-  const handleChange = (val: any) => console.log(val);
+  const [error, setError] = useState(
+    "Du må velge en aldersgruppe før du kan gå videre.",
+  );
+  const handleChange = (val: any) => {
+    console.log(val);
+    setError(val ? "" : "Du må velge en aldersgruppe før du kan gå videre.");
```

Comment:
Det er vel ikke mulig å få inn en falsy `val`, eller?
```suggestion
    setError("");
```

Reply:
Ah ja kan ikke unselecte radio..